toolkit/templates/layouts/_base_page.html

```diff
@@ -1,7 +1,8 @@
 {% extends "govuk/govuk_template.html" %}
 
 {% block head %}
-  {% block custom_meta %}{% include "toolkit/custom-dimensions.html" %}{% endblock %}
+  {% block custom_meta %}{% endblock %}
+  {% include "toolkit/custom-dimensions.html" %}
```

Comment:
Now that this is a thing we include from the base template, it should probably go into that `layouts` folder? Even though obviously it doesn't "lay anything out" :-o